File: desafio-02/patriciafelixx/javascript/numerosPrimos.js
const numerosPrimos = (x, y) => {
    for (let n = x; n <= y; n++) {
        let c = 0;
        for (let i = n; i >= 1; i--) {
            if (n % i == 0) {
                c++;
            }
        }
        c == 2 && console.log(n);
    }
}

numerosPrimos(1, 100);

File: desafio-02/evertonramos/java/NumerosPrimos.java
import java.util.ArrayList;
import java.util.List;

public class NumerosPrimos {

    public static void main(String[] args) {
        List<Integer> primos = new ArrayList<>();

        primos.add(2); // primeiro número primo

        for (int numero = 3; numero <= 10000; numero += 2) {
            for (Integer primo : primos) {
                if (numero % primo == 0) {
                    break; // não é primo
                }

                if ((double) numero / primo < primo) { // primo!
                    primos.add(numero);

                    break;
                }
            }
        }

        System.out.println(primos);
    }
}
